backend/app/services/integration_service.py

```
@@ -0,0 +1,230 @@
+    async def get_integrations(self, user_id: UUID) -> List[IntegrationResponse]:
+        """Get all integrations for a user."""
+        try:
+            result = await self.supabase.table("organization_integrations")\
+                .select("*")\
+                .eq("user_id", str(user_id))\
+                .execute()
+
+            return [IntegrationResponse(**item) for item in result.data]
+
+        except Exception as e:
+            logger.error(f"Error getting integrations: {str(e)}")
```

Ruff (0.14.0)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)